runtime/src/util/map.rs

```diff
@@ -153,25 +153,24 @@ where
         // wrapped in a hamt::Error::Dynamic.
         F: FnMut(K, &V) -> Result<(), ActorError>,
     {
-        match self.hamt.for_each(|k, v| {
-            let key = K::from_bytes(k).context_code(ExitCode::USR_ILLEGAL_STATE, "invalid key")?;
-            f(key, v).map_err(|e| anyhow!(e))
-        }) {
```

Comment:
Clippy was unhappy about all the code in the `match` part of the match statement, so I just got rid of the match entirely.

Reply:
this is probably a good call though, I find this kind of complex conditional construct puts unnecessary pressure on my mental stack